### DISCUSSION TITLE
We want to hear your ideas!

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#15] [Feature Requests]
Do you have ideas for improving the Deepgram experience or how to use Deepgram? We want to hear them!

We'll get the conversation started. Wouldn't it be cool to develop an app that helps kids learn to read or to practice their spelling words with Deepgram?

**Comment by bsixto** [comment]
Idea: 
Add a **noise detector**... if the speaker is in a noisy environment, it could be interesting to detect it (street, car, home-tv, family, ...).

**Reply by BekahHW** [reply]
That's really interesting! I like it when television shows have closed captioning for sound effects and background noise. It helps to better understand the context and tone of the scene.

**Comment by bsixto** [comment]
idea:
Set the time between the start of the recognition and the first word... by this way you can measure the user's **reactivity**...

**Reply by BekahHW** [reply]
ohhhh. That sounds like a cool hackathon project.

**Reply by michaeljolley** [reply]
Interesting. We kind of have that already. When you get a transcript back, the API returns a `words` array of every word it identified. Each of those words include a `start` and `end`. Using that first `word` object (which looks like below), you'll know the exact timestamp that the first word occurs.

```js
{
    "word": "string",
    "start": 0,
    "end": 0,
    "confidence": 0
} 
```
